trunk/SUAVE/Analyses/Mission/Segments/Single_Point/Set_Speed_Set_Altitude.py

```diff
@@ -1,7 +1,7 @@
 # Set_Speed_Set_Altitude.py
 #
 # Created:  Mar 2017, T. MacDonald
-# Modified: 
+# Modified: May 2017, T. MacDonald
```

Comment:
Yeah, that's an important change. However, since it's already in develop let's not change that.

Now that I look at this file though... there is initialization outside a method again in the class defaults. If you fix that, you can update the modified.